flocker/node/_docker.py

```diff
@@ -29,6 +30,9 @@
     RestartNever, RestartAlways, RestartOnFailure, pset_field, pvector_field)
 
 
+logger = logging.getLogger(__name__)
```

Comment:
See the change from the last two commits (mainly 88f9437) to see what was removed - the nice thing was that this was in the test file, so debugging output did not display during non-test operation.